### PR TITLE
Add Eurus mainnet network

### PR DESCRIPTION
**Pull request by EU99**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Your explorer does not seem to be EIP3091 compatible  - please change to `none`

**Comment by ligi** [comment]
you changed it in the wrong place - etherscan is compatible - your exporere is not ..-)

**Comment by EU99** [comment]
> you changed it in the wrong place - etherscan is compatible - your exporere is not ..-)

Sorry for the improper updates. Reverted and submitted again. :-P

File: _data/chains/eip155-1008.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "infoURL":"https://eurus.network",
+    "shortName":"eun",
+    "chainId":1008,
+    "networkId":1008,
+    "icon":"eurus",
+    "explorers":[
+       {
+          "name":"eurusexplorer",
+          "url":"https://explorer.eurus.network/",
```
Comment (ligi) [inline _data/chains/eip155-1008.json:24]:
```suggestion
          "url":"https://explorer.eurus.network",
```

File: _data/chains/eip155-1008.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "name":"Eurus Mainnet",
+    "chain":"EUN",
+    "network":"eurus",
+    "rpc":[
+       "https://mainnet.eurus.network/"
+    ],
+    "faucets":[
+
```
Comment (ligi) [inline _data/chains/eip155-1008.json:9]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please also remove the trailing slash to make the CI happy

**Review by ligi — APPROVED** [review]

**Pull request merged**